airmozilla/main/views.py

```diff
@@ -1114,6 +1115,90 @@ def item_pubdate(self, event):
         return event.start_time
 
 
+def related_content(request, slug):
+    event = get_object_or_404(Event, slug=slug)
+
+    es = pyelasticsearch.ElasticSearch(settings.RELATED_CONTENT_URL)
+    mlt_query = {
+        "more_like_this": {
+            "fields": ["title", "tags", "channels"],
```

Comment:
The idea is to not search for events that are on the main channel. It doesn't add any value to the search.